src/PhoneNumberInput/PhoneNumberInput.tsx

```diff
@@ -341,10 +344,17 @@ const StyledPhoneNumberWrapper = styled.div<StyledPhoneNumberWrapperProps>(
   })
 )
 const PrefixDropdownTrigger = styled.div(css(styles.dropdownTrigger) as any)
-const StyledPhoneNumberInput = styled.input<Props>(
+interface StyledPhoneNumberInputProps extends Props {
+  paddingRight: number
+}
+
+const StyledPhoneNumberInput = styled.input<StyledPhoneNumberInputProps>(
   css(styles.phoneNumberInput as SystemStyleObject) as any,
   variant({ variants }),
-  `border: none;`
+  `border: none;`,
+  ({ paddingRight }: StyledPhoneNumberInputProps) => ({
+    paddingRight: `${paddingRight * 4}px`
```

Comment:
You shouldn't multiply by 4 directly here ... it is a different behaviour than the normal size multiplication for other props.

Either use the css function from the styled system or find the real problem why this isn't automatically mapped to the theme  in the first place

Reply:
I'll look into how we could do this without explicit multiplication

Reply:
> ![image](https://private-user-images.githubusercontent.com/31573322/312393807-051a7562-cb9d-4dff-80ae-2689f8aeb6af.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAzMjM5OTMsIm5iZiI6MTcxMDMyMzY5MywicGF0aCI6Ii8zMTU3MzMyMi8zMTIzOTM4MDctMDUxYTc1NjItY2I5ZC00ZGZmLTgwYWUtMjY4OWY4YWViNmFmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzEzVDA5NTQ1M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQ0ODI4YTQyYWIxODk0NWRiOWU3Y2FhODQ5NmNhNGZlNWYzMmMzMGZjOWY4OWVmZmFjMjdiNTUzZWNiZGEyYzcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.weT1ew7Jqy9aDrVwG8DEMuD0nk5omUHiemOIdT3vwqI)

Nice, ai mai gasit un loc unde trebuie schimbat

Reply:
🤡

Reply:
c5655fcd1277b61721b0c8519eeb6b3adb3ba557